CONTRIBUTING.md

```diff
@@ -1,9 +1,18 @@
 ## Contributing
-GeoBlacklight is a collaborative open source projects where contributions are :sparkles:welcome:sparkles:. This contributing guide is borrowed in part from the [Samvera Contributing Guide](https://github.com/samvera/hydra/blob/master/CONTRIBUTING.md) and the [Blacklight Contributing Wiki](https://github.com/projectblacklight/blacklight/wiki/Contributing-to-Blacklight).
+GeoBlacklight is a collaborative, open source project where contributions are :sparkles:welcome:sparkles:. This contributing guide is borrowed in part from the [Samvera Contributing Guide](https://github.com/samvera/hydra/blob/master/CONTRIBUTING.md) and the [Blacklight Contributing Wiki](https://github.com/projectblacklight/blacklight/wiki/Contributing-to-Blacklight).
+
+### Join the community
+See the [community page](https://geoblacklight.org/community/) on our website for information about our community calendar, slack, volunteer opportunities, and more.
+
+If youhave questions, email the [GeoBlacklight Community Google Group](https://groups.google.com/g/geoblacklight-community) at [geoblacklight-community@googlegroups.com](mailto:geoblacklight-community@googlegroups.com).
 
 ### Who can contribute?
 **Anyone** is welcome to contribute to GeoBlacklight. We follow a set of contribution practices to maintain a technically sustainable and stable software project for everyone.
 
+### Who are the GitHub Developers Team?
+[GeoBlacklight Developers Team](https://github.com/orgs/geoblacklight/teams/geoblacklight-developers)
+<!--- More info needed! --->
+
 ### Reporting issues
```

Comment:
Suggest using a level 2 heading (##)

Reply:
I moved this in a commit I'm about to push, let me know what you think after that is in place. I moved this section down to where it's most relevant.